### DISCUSSION TITLE
Flet’s Approach in the Conversion of Flutter widgets/libraries into python classes

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#361] [Q&A]
Is it possible to describe flet's approach in the conversion of Flutter widgets/libraries into Python classes ? 

_Don't be afraid to go detaily/in-depth, cause this is a very interesting Topic._

**Comment by FeodorFitsner** [comment]
Sure, I'm working on a new contribution guide.
